contracts/authorization/src/contract.rs

```diff
@@ -693,12 +704,47 @@ fn process_processor_callback(
         ));
     }
 
-    // Updated the result
+    // Update the result
     callback.execution_result = execution_result;
-
     PROCESSOR_CALLBACKS.save(deps.storage, execution_id, &callback)?;
 
+    // Reduce the current executions for the label
+    CURRENT_EXECUTIONS.update(
+        deps.storage,
+        callback.label.clone(),
+        |current| -> StdResult<_> { Ok(current.unwrap_or_default().saturating_sub(1)) },
```

Comment:
This looks weird, default for u64 is 0, seems like if its 0 it should error, no?
Can you explain a little bit more about it?

Reply:
It won't be 0, if we are processing a callback it will be at least 1 or more. This tracks how many are being executed for each label.

Reply:
If it should never be 0, we should test for it here.

Reply:
Sure, added it